### ISSUE TITLE
Secondary Structure Prediction with RibonanzaNet-SS?

### ISSUE THREAD

**Issue by chaitjo** [MarksLab-DasLab/RNAGym#14]
Have the team benchmarked RibonanzaNet-SS vs. all the other secondary structure predictors? The Ribonanza paper claimed that this model was a significant gain in terms of performance.

Additionally, there is also a secondary structure finetuned version of RiNaLMo. I would be very interested in independent benchmarking of these models, and perhaps how they compare to the predicted secondary structure from AlphaFold 3 or similar tools (just doing DSSR on the 3D prediction)

**Comment by murfalo** [comment]
Hello, and so sorry for the delay as the team was away for the holidays!  With respect to your questions:

> Have the team benchmarked RibonanzaNet-SS vs. all the other secondary structure predictors? The Ribonanza paper claimed that this model was a significant gain in terms of performance.

We report the performance of RibonanzaNet-SS in the supplement (Table A12).  RibonanzaNet-SS had substantial overlap with our test set, so we retested all the models + RibonanzaNet-SS on the intersection of our test set and RibonanzaNet-SS's test set.  On this restricted test set, RibonanzaNet-SS appears to struggle with DMS data but excel at 2A3.  We are currently exploring more objective methods of quantifying supervised method performance relative to similarity to training and hope to include this in a future edition of RNAGym.

> Additionally, there is also a secondary structure finetuned version of RiNaLMo. I would be very interested in independent benchmarking of these models, and perhaps how they compare to the predicted secondary structure from AlphaFold 3 or similar tools (just doing DSSR on the 3D prediction).

The secondary structure finetuned version of RiNaLMo will be great to include in a future release.   The comparison you bring up is a great point as well, and we are actively working on testing the secondary structure models on our 3D dataset to enable direct comparison with the 3D models.

Hopefully that answers all your questions.  Please reopen the issue if otherwise.  And thank you for the excellent feedback!
